app/assets/javascripts/admin/utils/services/errors_parser.js.coffee

```diff
@@ -2,16 +2,25 @@
 angular.module("admin.utils").factory "ErrorsParser", ->
   new class ErrorsParser
     toString: (errors, defaultContent = "") =>
+      return defaultContent unless errors?
+
+      errorsString = ""
       if errors.length > 0
         # it is an array of errors
-        errorsString = error + "\n" for error in errors
+        errorsString = this.arrayToString(errors)
       else
         # it is a hash of errors
         keys = Object.keys(errors)
-        errorsString = ""
         for key in keys
-          errorsString += error for error in errors[key]
+          errorsString += this.arrayToString(errors[key])
```

Comment:
(But need to add the trailing newline after each key here if going to use `Array#join`.)

Reply:
thanks for the heads up, could have forgotten this :+1: